docs/src/main/sphinx/admin/properties-optimizer.rst

```diff
@@ -116,3 +116,14 @@ the maximum number of joins that can be reordered at once.
 
 Reduces number of rows produced by joins when optimizer detects that duplicated
 join output rows can be skipped.
+
+``optimizer.table-scan-node-partitioning-min-bucket-to-task-ratio``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``double``
+* **Default value:** ``0.5``
+
+Specifies minimal bucket to task ratio that has to be exceeded in order
+to use table scan node partitioning. When there is not enough buckets
```

Comment:
When there is not enough buckets... -> When the number of buckets is small compared to the number of table scan tasks then the table scan will be ...